llvm/include/llvm/IR/IntrinsicsLoongArch.td

```diff
@@ -621,22 +621,22 @@ foreach inst = ["vfcmp_caf_d", "vfcmp_cun_d", "vfcmp_ceq_d", "vfcmp_cueq_d",
 // LSX load/store
 def int_loongarch_lsx_vld
   : VecInt<[llvm_v16i8_ty], [llvm_ptr_ty, llvm_i32_ty],
-           [IntrReadMem, IntrArgMemOnly, ImmArg<ArgIndex<1>>]>;
+           [IntrWillReturn, IntrReadMem, IntrArgMemOnly, ImmArg<ArgIndex<1>>]>;
```

Comment:
Use default attrs here as well? From a quick search, these are normal (non-synchronizing) vector loads.

Reply:
Originally avoided since these inherit from `VecInt`, but created class `DefaultAttrsVecInt` which inherits from `DefaultAttrsIntrinsic` instead of `Intrinsic` which should accomplish this.